kratos/linear_solvers/amgcl_solver.h

```diff
@@ -115,7 +115,7 @@ AMGCLScalarSolve(
     std::tie(rIterationNumber, rResidual) = solve(rB, rX);
 
     if(verbosity_level > 1 )
-        std::cout << "AMGCL Memory Occupation (MB): " << amgcl::backend::bytes(solve) << std::endl;
+        std::cout << "AMGCL Memory Occupation (kB): " << amgcl::backend::bytes(solve)/1024 << std::endl;
```

Comment:
There is also `amgcl::human_readable_memory()` available which will give you the most appropriate units with 2 digits precision:

https://github.com/ddemidov/amgcl/blob/d1f8387a05ccc409d57505460d5fde27637ce902/amgcl/util.hpp#L310-L320